File: BookMyShow.Web/bookMyShow/src/app/shared/models/theatre.ts
export class Theatre {
    id: number;
    name: string;
    rows: number;
    columns: number;
    movieIds: number[];
    showTime: string;
    location: string;
    ticketPrice: number;
    constructor(id: number, name: string, location: string, movieIds: number[], rows: number, columns: number, showTime: string, ticketPrice: number) {
        this.id = id;
        this.name = name;
        this.rows = rows;
        this.columns = columns;
        this.showTime = showTime;
        this.ticketPrice = ticketPrice;
        this.movieIds = movieIds;
        this.location = location;
    }
}

File: BookMyShow.API/BookMyShow.Data/DataModels/Theatre.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BookMyShow.Data.DataModels
{
    [Table("Theatre")]
    public class Theatre
    {
        [Key]
        [Column("Id")]
        public int Id { get; set; }

        [Column("Name")]
        public string Name { get; set; }

        [Column("Rows")]
        public int Rows { get; set; }

        [Column("Columns")]
        public int Columns { get; set; }

        [Column("MovieIds")]
        public string MovieIds { get; set; }

        [Column("ShowTime")]
        public string ShowTime { get; set; }

        [Column("TicketPrice")]
        public float TicketPrice { get; set; }

        [Column("Location")]
        public string Location { get; set; }

    }
}
